src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,19 @@
+
+import { SearchResponse } from '../types/SearchApiTypes';
+
+export const getDataDisplay = async (): Promise<SearchResponse> => {
+  try {
+    const response = await fetch('/data-display', {
+      method: 'GET',
+    });
+
+    if (!response.ok) {
+      throw new Error('Failed to fetch data display');
+    }
+
+    const data = await response.json();
+    return data as SearchResponse;
+  } catch (error) {
+    throw new Error(`Error in getDataDisplay API: ${error.message}`);
+  }
```

Comment:
`error` object should be with type: 
`error: any`

And we don't have error schema defined. So, don't print `error.message`.